### DISCUSSION TITLE
Alpine js Tabs problem

### DISCUSSION THREAD

**Discussion by jasha1974** [Power-Components/livewire-powergrid#183] [Q&A]
hi, so i have another problem, if using tabs, no content visible after insert livewire component. Could you check code bellow ?

Thanx a lot Slavik J

<div class="container mx-auto" x-data="{ tab: 'tab1' }">
                <ul class="flex justify-center  ">
                    <li class="-mb-px mr-1">
                        <a
                            class="inline-block rounded-t py-2 px-4 font-semibold hover:text-blue-800"
                            :class="{ 'border-indigo-400 border-b' : tab === 'tab1' }"
                            href="#"
                            @click.prevent="tab = 'tab1'"
                        >{{ __('Users') }}</a>
                    </li>
                    <li class="-mb-px mr-1">
                        <a
                            class="inline-block py-2 px-4 text-blue-500 hover:text-blue-800 font-semibold"
                            :class="{ 'border-indigo-400 border-b' : tab === 'tab2' }"
                            href="#"
                            @click.prevent="tab = 'tab2'"
                        >{{ __('Orders') }}</a>
                    </li>
                    <li class="-mb-px mr-1">
                        <a
                            class="inline-block py-2 px-4 text-blue-500 hover:text-blue-800 font-semibold"
                            :class="{ 'border-indigo-400 border-b' : tab === 'tab3' }"
                            href="#"
                            @click.prevent="tab = 'tab3'"
                        >{{ __('Import') }}</a>
                    </li>
                </ul>
                <div class="content px-1 py-1 pt-4">

                    <div x-show="tab === 'tab1'">

                        <div class="pb-8">
                            <livewire:user-table/>
                        </div>

                    </div>

                    <div x-show="tab === 'tab2'">

                    </div>

                    <div x-show="tab === 'tab3'">

                        <div class="py-2">
                            <x-inputX type="file" placeholder="Select file ..." wire:model="importFile" />
                            <x-buttonX wire:click="import">Import csv data</x-buttonX>
                            <span class="pl-2">{{ $status }}</span>
                        </div>

                    </div>

                </div>
            </div>

**Comment by luanfreitasdev** [comment]
Hi, could you give more details? Maybe a video showing how you did it what happens

**Reply by jasha1974** [reply]
thanks for answer, so i using different solutions, pls, close thread
